csharp/src/Google.Protobuf/Reflection/FieldDescriptor.cs

```diff
@@ -168,14 +168,16 @@ public bool IsMap
             get { return fieldType == FieldType.Message && messageType.Proto.Options != null && messageType.Proto.Options.MapEntry; }
         }
 
-        // TODO(jonskeet): Check whether this is correct with proto3, where we default to packed...
-
         /// <summary>
         /// Returns <c>true</c> if this field is a packed, repeated field; <c>false</c> otherwise.
         /// </summary>
         public bool IsPacked
```

Comment:
We'll need this for dynamic message, and anything else that uses reflection
to work out what to write. It felt important enough to expose publicly, and
get right...
On 8 Aug 2015 20:10, "Jan Tattermusch" <notifications@github.com> wrote:

> In csharp/src/Google.Protobuf/Reflection/FieldDescriptor.cs
> <https://github.com/google/protobuf/pull/699#discussion_r36580897>:
>
> > @@ -168,14 +168,16 @@ public bool IsMap
> >              get { return fieldType == FieldType.Message && messageType.Proto.Options != null && messageType.Proto.Options.MapEntry; }
> >          }
> >
> > -        // TODO(jonskeet): Check whether this is correct with proto3, where we default to packed...
> > -
> >          /// <summary>
> >          /// Returns <c>true</c> if this field is a packed, repeated field; <c>false</c> otherwise.
> >          /// </summary>
> >          public bool IsPacked
>
> looks like this is only used in WireFormat.MakeTag(int, FieldDescriptor).
> The generated code already has tags inlined, so what is this going to be
> used for? DynamicMessage? In that case we should make sure we fail early
> once someone tries to push us a descriptor for proto2 message (but I think
> we are already doing that).
>
> Just wondering if it wouldn't be fine to just hide this field.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/protobuf/pull/699/files#r36580897>.
>